roles/openshift_logging/templates/es.j2

```diff
@@ -64,7 +67,7 @@ spec:
               value: "logging-{{es_cluster_name}}"
             -
               name: "INSTANCE_RAM"
-              value: "{{openshift_logging_es_instance_ram}}"
+              value: "{{openshift_logging_es_memory_limit}}"
```

Comment:
since our run.sh used INSTANCE_RAM and expected it to not have an 'i' at the end, it should probably be updated too or at least verify that it still works as expected.